smart/plugin.go

```diff
@@ -213,10 +217,11 @@ func (sc *SmartCollector) CollectMetrics(mts []plugin.MetricType) ([]plugin.Metr
 			}
 		} else {
 			// Single disk requested
-
 			collected, result, err := sc.DiskMetrics(ns, t, disk, attribute_path, buffered_results, errs)
 			if err != nil {
-				sc.logger.Error(fmt.Sprintf("Error collecting SMART %s data on %s disk: %#+v", attribute_path, disk, err))
+				eStr := fmt.Sprintf("Error collecting SMART %s data on %s disk: %#+v", attribute_path, disk, err)
+				sc.logger.Error(eStr)
```

Comment:
logging single error here but on the line 235, it logs all the errors. It seems that duplicated errors being logged?